templates/main.php

```diff
@@ -51,35 +49,38 @@ class="collection"
+                        <button></button>
+                    </span>
+                    <div class="app-navigation-entry-menu" ng-show="calendar.writable">
+                        <ul>
+                            <li>
+                                <a ng-click="startEdit(calendar)">
+                                    <span class="icon-rename svg"></span>
+                                    <span><?php p($l->t('Edit')); ?></span>
+                                </a>
+                            </li>
+                            <li>
+                                <a ng-click="showCalDAVUrl(calendar)">
+                                    <span class="icon-public svg"></span>
+                                    <span><?php p($l->t('Link')); ?></span>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="{{calendar.exportUrl}}" download="{{calendar.uri}}.ics">
+                                    <span class="icon-download svg"></span>
+                                    <span><?php p($l->t('Download')); ?></span>
+                                </a>
+                            </li>
+                            <li confirmation="delete(calendar)" confirmation-message="deleteMessage(calendar)">
+                            </li>
+                        </ul>
+                    </div>
                 </div>
-                <div class="app-navigation-entry-menu" ng-show="calendar.writable">
-                    <ul>
-                        <li>
-                            <span ng-click="startEdit(calendar)">
-                                <img class="icon-rename svg" src="<?php p(image_path('core', 'actions/rename.svg'))?>"/>
-                                <span><?php p($l->t('Edit')); ?></span>
-                            </span>
-                        </li>
-                        <li>
-                            <span ng-click="showCalDAVUrl(calendar)">
-                                <img class="icon-public svg" src="<?php p(image_path('core', 'actions/public.svg'))?>"/>
-                                <span><?php p($l->t('Link')); ?></span>
-                            </span>
-                        </li>
-                        <li>
-                            <a href="{{calendar.exportUrl}}" download="{{calendar.uri}}.ics">
-                                <img class="icon-download svg" src="<?php p(image_path('core', 'actions/download.svg'))?>"/>
-                                <span><?php p($l->t('Download')); ?></span>
-                            </a>
-                        </li>
-                        <li confirmation="delete(calendar)" confirmation-message="deleteMessage(calendar)">
-                        </li>
-                    </ul>
+                <div class="app-navigation-entry-utils-counter">
```

Comment:
Should be first. Before the `app-navigation-entry-menu` and the `app-navigation-entry-utils-menu-button` button.

And it's empty on my installation, therefore moving the menu icon to the left. 
![capture d ecran_2017-02-20_05-36-31](https://cloud.githubusercontent.com/assets/14975046/23112178/8d291c9c-f72e-11e6-8e65-3a31fc2eb345.png)